rotkehlchen/history/events.py

```diff
@@ -477,14 +478,16 @@ def query_asset_movements(
 
     def get_history(
             self,
+            report_id: int,
             start_ts: Timestamp,
             end_ts: Timestamp,
             has_premium: bool,
     ) -> HistoryResult:
         """Creates trades and loans history from start_ts to end_ts"""
         self._reset_variables()
         step = 0
-        total_steps = len(self.exchange_manager.connected_exchanges) + NUM_HISTORY_QUERY_STEPS_EXCL_EXCHANGES  # noqa: E501
+        total_steps = len(
```

Comment:
don't do that. Why do that?

Reply:
These are coming from my editor reformatting the file, I will investigate and turn that off